### ISSUE TITLE
loading scannet "keyerror"

### ISSUE THREAD

**Issue by ooodragon94** [plusmultiply/mprm#6]
hello!
Want to ask for some help while loading .ply data
I tried to load .ply data with "read_ply" function and it gave me keyerror

```
<ipython-input-17-c43e6f65c260> in parse_header(plyfile, ext)
     40             line = line.split()
     41             print(line)
---> 42             properties.append((line[2].decode(), ext + ply_dtypes[line[1]]))
     43 
     44     return num_points, properties

KeyError: b'list'
```

any help on this? thanks a lot!

**Comment by starrism-yu** [comment]
Hi, Have you solved this problem? I have the same problem.
 Can you give me some help?
thanks a lot !!!

**Comment by ooodragon94** [comment]
No.. I have not. It was too far away for me and I was merely able to debug anything that time

However, looking at the error function it might be the binary file that is causing error
you might want to see if it is line[2].decode() that is making the error or ply_dtypes[line[1]] that is making the error 
and start debugging from there
probably will work without b? I'm not so sure.
